subprojects/using-build-cache/build.gradle.kts

```diff
@@ -5,7 +5,7 @@ plugins {
 
 guide {
     repositoryPath.set("gradle-guides/using-build-cache")
-    minimumGradleVersion.set("5.0")
+    minimumGradleVersion.set("6.0")
```

Comment:
What's this used for?

Reply:
All the links to the docs go to this Gradle version. We don't link to `current` from guides.

Reply:
👍